langworld_db_data/adders/listed_value_adder.py

```diff
@@ -89,6 +131,57 @@ def _add_to_inventory_of_listed_values(
 
         return id_of_new_value
 
+    @staticmethod
+    def _get_indices_and_their_line_numbers_in_features_listed_values(
+        rows: list[dict[str, str]],
+        feature_id: str,
+        new_value_en: str,
+        new_value_ru: str,
+    ) -> list[dict[str, int]]:
+
+        value_indices_to_inventory_line_numbers: list[dict[str, int]] = []
+        """List of dictionaries, each mapping value index to line number in features_listed_values,
+        ex. [{"index": 1, "line number": 4}, {"index": 2, "line number": 5}, {"index": 3, "line number": 6}].
+        Contains all indices and line numbers of values with given feature_id.
```

Comment:
```suggestion
        """List of dictionaries, each mapping value index to line number in the part of inventory of listed values
        pertaining to the given feature ID,
        e.g. [{"index": 1, "line number": 4}, {"index": 2, "line number": 5}, {"index": 3, "line number": 6}].
```
better to state in the very first phrase that it's done for _one feature ID_.

Reply:
Done! In this docstring, there is a line "`Contains all indices and line numbers of values with given feature_id.`", I saved it for good measure, too

Reply:
TIL: "for good measure"